ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseMapper.java

```diff
@@ -79,6 +90,9 @@ public CCDCase to(Claim claim) {
                 .build())
         );
 
+        claim.getClaimantResponse()
+            .ifPresent(claimantResponse -> builder.claimantResponse(claimantResponseMapper.to(claimantResponse)));
```

Comment:
Generally preferred to make lambdas atomic and chained.
e.g. if you agree: `claim.getClaimantResponse().map(claimandResponseMapper::to).ifPresent(builder::claimantResponse)`